client/test/server.go

```diff
@@ -37,6 +37,7 @@ type updateType struct {
 	Unauthorized bool
 	Called       bool
 	Current      *client.CurrentUpdate
+	ControlMap   interface{}
```

Comment:
I sense there is something I don't understand about why `interface{}` is being used, but I'll ask anyway: Why `interface{}` and not a proper type?

Reply:
Because the 'real' type `UpdateResponse` is now defined in `app/mender.go`, and importing it will cause a cycle :(

Reply:
Oh right, it's what the talked about, I see it now.

But seeing these pure interfaces everywhere, this really isn't pretty... Maybe we should just bite the bullet and make these two different structs. It's not super nice either, but IMHO the pure `interface{}`s are worse. They're hard to read and understand, and they open up for very easy mistakes later on.